### PR TITLE
add contributing guidelines and update github stuff

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
added CONTRIBUTING.md for the PR template and for the wiki to link to
its linked at the top of the template now to make it easy to find probably.

## Why / Balance
new contributors didnt get explicitly told any deltav conventions :trollface:

also old MARKERS.md link didnt exist :trollface:

## Technical details
no

## Media
no

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
no

**Changelog**
no cl no fun

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,23 +10,22 @@
...
 ## Media
-<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
-Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
+<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
+Small fixes/refactors are exempt. -->
 
 ## Requirements
 <!-- Confirm the following by placing an X in the brackets [X]: -->
-- [ ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
+- [ ] I have tested all added content and changes.
```
Comment (MilonPL) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: CONDEMNS BILLIONS

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
+<!-- If you are new to the Delta-V repository, please read the [Contributing Guidelines](https://github.com/DeltaV-Station/Delta-v/blob/master/CONTRIBUTING.md) -->
```
Comment (MilonPL) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: We could just use the wiki if you want to, we have the means

File: CONTRIBUTING.md
Comment (MilonPL) [inline CONTRIBUTING.md:1]: Couple of notes from me:
- Always ask the map maintainer before making changes to a map.
- Do not use GitHub's web editor to create PRs. PRs submitted through the web editor may be closed without review. (no more webedit ops, that's from wizden)
- Provide screenshots or videos showing that your changes were tested in-game. 
- Double-check your diff on GitHub before submitting: look for unintended commits or changes and remove accidental whitespace or line-ending changes.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (deltanedas) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: you can just check it anyway :trollface:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (deltanedas) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: i originally wanted to put it on the wiki but mediawiki is so ass and doesnt even have multitline code blocks

Comment (MilonPL) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: i can install SyntaxHighlight or something if you want me to

File: CONTRIBUTING.md
Comment (deltanedas) [reply CONTRIBUTING.md:1]: third one is for template (and already exists)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (MilonPL) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: pro
image

Comment (deltanedas) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: still good to have here as well so you can read it offline or if the wiki dies (not that it ever would :trollface:)

Comment (MilonPL) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: yeah just mirror it to the wiki, sucks to have it maintained in two different places but it's not like there's going to be changes to this often

**Comment by Stop-Signs** [comment]
Finally I can stop getting audited over the silly #

**Comment by deltanedas** [comment]
its being enforced already :trollface:

**Comment by MilonPL** [comment]
> its being enforced already :trollface:

i was waiting on you to change the link and that never came...

**Pull request merged**